deployments/types/rococo-wococo.json

```diff
@@ -0,0 +1,52 @@
+{
+	"--1": "Rococo Types",
+	"RococoBalance": "u128",
+	"RococoBlockHash": "H256",
+	"RococoBlockNumber": "u32",
+	"RococoHeader": {
+		"parent_Hash": "RococoBlockHash",
+		"number": "Compact<RococoBlockNumber>",
+		"state_root": "RococoBlockHash",
+		"extrinsics_root": "RococoBlockHash",
+		"digest": "RococoDigest"
+	},
+	"RococoDigest": {
+		"logs": "Vec<RococoDigestItem>"
+	},
+	"RococoDigestItem": {
+		"_enum": {
+			"Other": "Vec<u8>",
+			"AuthoritiesChange": "Vec<AuthorityId>",
+			"ChangesTrieRoot": "RococoBlockHash",
+			"SealV0": "SealV0",
+			"Consensus": "Consensus",
+			"Seal": "Seal",
+			"PreRuntime": "PreRuntime"
+		}
+	},
+	"--2": "Wococo Types",
+	"WococoBalance": "u128",
+	"WococoBlockHash": "H256",
+	"WococoBlockNumber": "u32",
+	"WococoHeader": {
+		"parent_Hash": "WococoBlockHash",
+		"number": "Compact<WococoBlockNumber>",
+		"state_root": "WococoBlockHash",
+		"extrinsics_root": "WococoBlockHash",
+		"digest": "WococoDigest"
+	},
+	"WococoDigest": {
+		"logs": "Vec<WococoDigestItem>"
+	},
+	"WococoDigestItem": {
+		"_enum": {
+			"Other": "Vec<u8>",
+			"AuthoritiesChange": "Vec<AuthorityId>",
+			"ChangesTrieRoot": "WococoBlockHash",
+			"SealV0": "SealV0",
+			"Consensus": "Consensus",
+			"Seal": "Seal",
+			"PreRuntime": "PreRuntime"
+		}
+	}
```

Comment:
This is most likely not necessary - it was only really needed for Millau, because the regular `Header` type has smaller hash size hardcoded.

While this is not wrong obviously, I think it might make sense to keep the types minimal, so that it's easier to integrate them in the future with polkadot.js/api core types.